src/ari_sxn_simbeamline/xrt_sim.py

```diff
@@ -0,0 +1,131 @@
+import sys
+
+# sys.path.append('/Users/jieminli/PycharmProjects/Beamline_sim/')
+import numpy as np
+# from scipy import *
+# from pandas import DataFrame
+import matplotlib.pyplot as plt
+from pandas import DataFrame
+#################################################################
+# import xrt packages
+import xrt.backends.raycing as raycing
+import xrt.backends.raycing.run as rr
+import xrt.plotter as xrtp
+import xrt.runner as xrtr
+import xrt.backends.raycing.sources as rs
+from xrt.backends.raycing.physconsts import SIE0
+import xrt.backends.raycing.apertures as raa
+import xrt.backends.raycing.screens as rsc
+import xrt.backends.raycing.oes as roe
+import xrt.backends.raycing.materials as rm
+import xrt.backends.raycing.materials_compounds as rmComp
+
+#################################################################
+gold = rm.Material('Au', rho=19.3, table='Chantler Total', kind='mirror', name='Au')
+nickel = rm.Material('Ni', rho=8.908, table='Chantler Total', kind='mirror', name='Ni')
+genericGR = rm.Material('Ni', rho=8.908, table='Chantler total', kind='grating',
+                        name='generic grating', efficiency=[(1, 1), (-1, 1)])  # efficiency = 1
+
+
+#################################################################
+
+def buildBeamline(eV, sigma_eV, Nrays, HsltSz=4, VsltSz=4):
+    # build a properly aligned beamline
```

Comment:
Please use the standard structure for doc-strings, this will help later with documentation